apps/server/src/modules/tool/external-tool/uc/external-tool.uc.spec.ts

```diff
@@ -468,4 +481,77 @@ describe('ExternalToolUc', () => {
 			expect(externalToolService.deleteExternalTool).toHaveBeenCalledWith(toolId);
 		});
 	});
+
+	describe('getMetadataForExternalTool', () => {
+		describe('Authorization', () => {
+			const setupMetadata = () => {
```

Comment:
setup

Reply:
setup is used in the whole class, i need to refactor the whole test file then..